src/main/webapp/resources/js/pages/admin/components/statistics/BasicStats.jsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * This file renders the Basic Statistics component
+ * with functionality to view advanced statistics
+ * charts
+ */
+
+import React, { useContext, useState } from "react";
+import { Button, Card, Statistic } from "antd";
+import { SPACE_LG, SPACE_XS } from "../../../../styles/spacing";
+import AdvancedStatistics from "./AdvancedStatistics";
+import {
+  AdminStatisticsContext,
+  defaultTimePeriod,
+  statisticTypes,
+  timePeriodMap
+} from "../../../../contexts/AdminStatisticsContext";
+
+export default function BasicStats() {
+  const [statsChartView, setStatsChartView] = useState(null);
+
+  const { adminStatisticsContext } = useContext(
+    AdminStatisticsContext
+  );
+
+  return (
+    <div className="t-statistics">
+      <div style={{marginBottom: SPACE_LG}} className="t-stats-basic">
+        <div style={{display: "flex",
+          justifyContent: "flex-start", flexWrap: "wrap"}}>
+            <Card style={{width: 240, marginRight: SPACE_XS}}>
```

Comment:
Take a look at `Row` and `Col` from Ant Design.  You get a lot of this with proper height separations and media queries. Right now your cards stack on touching each other

Reply:
I ended up adding marginBottom to the cards. The Row/Col broke the display and the buttons would overlap

Reply:
Updated in ff7ec157807d1b1bc99e6e65236ed77a1bcee291